code/obj/item/clothing/masks.dm

```diff
@@ -100,7 +100,7 @@
 	name = "gas mask"
 	desc = "A close-fitting mask that can filter some environmental toxins or be connected to an air supply."
 	icon_state = "gas_mask"
-	c_flags = SPACEWEAR | COVERSMOUTH | COVERSEYES | MASKINTERNALS | BLOCKSMOKE
+	c_flags = SPACEWEAR | COVERSMOUTH | COVERSEYES | MASKINTERNALS | BLOCKSMOKE | BLOCKCHOKE
```

Comment:
It's probably a better idea to keep the `BLOCKCHOKE` flag on helmets only (possibly with some rare exceptions). 
Why?
Because gas masks are very common; they spawn on racks, in certain closets and some jobs start with them. Paired with a helmet, that's a lot of free choke protection for little (or in some cases, zero) effort.

Reply:
Agreed. Perhaps there could be a non-internals mask that blocks choking at some point?

Reply:
~~Clown mask~~

Reply:
Hmm would it be a better idea to just remove the mask's protection from vampire bite entirely? Because currently it's in a weird place where cigarettes block vamp bite.
And I can't think of any other solution really. It would buff vamps slightly without buffing gas masks overall

Reply:
yeah

Reply:
ok, done